.github/workflows/pull-request-workflow.yml

```diff
@@ -0,0 +1,39 @@
+name: Pull Request Workflow
+on: [pull_request]
+jobs:
+  run-tests:
+    runs-on: macos-11
+    timeout-minutes: 15
+    steps:
+    - name: Cancel previous jobs
+      uses: styfle/cancel-workflow-action@0.6.0
+      with:
+        access_token: ${{ github.token }}
+    - name: Git checkout
+      uses: actions/checkout@v2.3.4
+      with:
+        ref: ${{ github.ref }}
+    - name: Setup Xcode
+      uses: maxim-lobanov/setup-xcode@v1
+      with:
+        xcode-version: '12.4'
+    - name: Setup ruby and bundler dependencies
+      uses: ruby/setup-ruby@v1.81.0
+      with:
+        bundler-cache: true
+    - name: Run pod install
+      run: bundle exec pod install --project-directory=Example
+    - name: Add SSH key
+      run: ssh-add - <<< "${{ secrets.SSH_KEY }}"
+    - name: Allow SSH fingerprinting
+      run: |
+        sudo defaults write com.apple.dt.Xcode IDEPackageSupportUseBuiltinSCM YES
+        rm ~/.ssh/id_rsa || true
+        for ip in $(dig @8.8.8.8 bitbucket.org +short); do ssh-keyscan bitbucket.org,$ip; ssh-keyscan $ip; done 2>/dev/null >> ~/.ssh/known_hosts || true 
+        for ip in $(dig @8.8.8.8 github.com +short); do ssh-keyscan github.com,$ip; ssh-keyscan $ip; done 2>/dev/null >> ~/.ssh/known_hosts || true
```

Comment:
https://stackoverflow.com/questions/58125659/github-actions-xcodebuild-fails-due-to-server-fingerprint

Reply:
What's the likelihood that this will need changing?

Reply:
No idea yet, but I'll tell you once my [order has arrived](https://www.amazon.co.uk/s?k=crystal+ball&ref=nb_sb_noss).

Reply:
hahah, fair point 😂